src/control/cmd/dmg/network.go

```diff
@@ -42,18 +42,17 @@ type NetCmd struct {
 // that match the given fabric provider.
 type networkScanCmd struct {
 	logCmd
+	cfgCmd
 	ctlInvokerCmd
 	hostListCmd
 	jsonOutputCmd
-	FabricProvider string `short:"p" long:"provider" description:"Filter device list to those that support the given OFI provider (default is the provider specified in daos_server.yml)"`
-	AllProviders   bool   `short:"a" long:"all" description:"Specify 'all' to see all devices on all providers.  Overrides --provider"`
+	FabricProvider string `short:"p" long:"provider" description:"Filter device list to those that support the given OFI provider or 'all' for all available (default is the provider specified in daos_server.yml)"`
 }
 
 func (cmd *networkScanCmd) Execute(_ []string) error {
 	ctx := context.Background()
-	req := &control.NetworkScanReq{
-		Provider: cmd.FabricProvider,
-	}
+	req := &control.NetworkScanReq{Provider: cmd.FabricProvider}
+
```

Comment:
Unintentional formatting change.  I can fix it if you want.

Reply:
Doesn't bother me as long as gofmt is happy with it.